users/views.py

```diff
@@ -117,11 +117,11 @@ def get(self, request, *args, **kwargs):
                 "last_name": user_profile.user.last_name,
                 "email": user_profile.user.email,
                 "telephone_number": str(user_profile.telephone_number),
-                "forwardingEmail": user_profile.forwarding_email
```

Comment:
you probably need to change this back to `forwarding_email` after Justins fix in PR #86